src/main/java/EOorg/EOeolang/EOtxt/EOtext$EOreplaced.java

```diff
@@ -54,8 +54,8 @@ public class EOtext$EOreplaced extends PhDefault {
      *
      * @param sigma Sigma
      * @todo #21:30min Current implementation using Java.
-     *  We should implement it only via EOLANG code. We can do
-     *  it by using reduce method
+     * We should implement it only via EOLANG code. We can do
```

Comment:
As I know `@todo` description must have a one more intendation than `@todo`. That's why there were spaces here.